added release notes up to 10.17.0.195 (#246)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/devices-10-17-0.md

<td>n/a</td>
</tr>

<tr>
<td>OPC UA</td>

> <td>If the namespace is not defined for a specific node, a missing error handler in the past prevented the scan operation to complete. This is now fixed by skipping nodes that cannot be read. The opcua-device-gateway logs further detail in such cases. The log level INFO contains the failure message and DEBUG contains the stack trace.</td>

Reviewer: ```suggestion
<td>If the namespace is not defined for a specific node, a missing error handler prevented the scan operation from completing. This is now fixed by skipping nodes that cannot be read. The opcua-device-gateway logs further detail in such cases. The log level INFO contains the failure message and the log level DEBUG contains the stack trace.</td>
```

Author: You reviewed it already once some time ago, slightly different, I will use this first reviewed version.
